apis/core/lib/domain/job/create.ts

```diff
@@ -76,12 +76,14 @@ export async function createPublishJob({
 
   const metadata = await store.getResource(project.dataset)
 
+  const targetGraph = project.isHiddenCube ? $rdf.namedNode(organization.publishGraph.value + '/hidden') : organization.publishGraph
```

Comment:
Still thinking this could be made explicitly part of the organisation resources in DB. Could make communication with VSHN easier.

If not, might at least wrap as a getter in the [`OrganizationMixin` class](https://github.com/zazuko/cube-creator/blob/master/packages/model/Organization.ts#L28-L74).